finite-state/shared/src/main/scala/fs2/data/mft/package.scala

```diff
@@ -0,0 +1,60 @@
+package fs2.data
+
+package object mft {
+
+  def dsl[InTag, OutTag](build: MFTBuilder[InTag, OutTag] => Unit): MFT[InTag, OutTag] = {
```

Comment:
Would love to see an overload for Scala 3 that takes a context function.

Reply:
:grimacing: OK